lineman/app/css/1_utilities.scss

```diff
@@ -109,11 +109,15 @@ label.label-with-details{
   padding: 10px;
 }
 
-.lmo-h1{
+@mixin lmoH1{
```

Comment:
Fair enough. 

I just went through and applied this pattern to all the button classes and it felt like a great improvement. See what you think: #2634 

I like the abstraction because it reminds me, 'hey this collection of style declarations is reused in a whole bunch of places'. In many places it meant I could remove all the utility classes from the view and only be left with a single semantic class names on each element, and all the style information relegated to a css file where it should be. (I only like to use utility classes in the view when we don't already have a css file for that block.)